### PR TITLE
39: add jolokia for zookeeper monitoring

### PR DESCRIPTION
**Pull request by ghost**
Jolokia support for Zookeeper monitoring was added. #39

### CODE REVIEW

**Comment by zrobgar** [comment]
👍

File: Dockerfile [hunk 1]
```diff
@@ -44,7 +46,6 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     && rm -rf /var/lib/apt/lists/* /root/.m2
 
 # Base part of dockerfile should not be modified not to rebuild image for 1 hour
-ENV JOLOKIA_VERSION="1.3.7"
```
Comment (antban) [inline Dockerfile]: What is the purpose of moving this variable up? The install here takes time and reordering commands will cause longer rebuild

File: Dockerfile [hunk 2]
```diff
@@ -57,6 +58,6 @@ COPY scm-source.json /scm-source.json
 WORKDIR ${HOME}
 USER ${USER}
 
-EXPOSE 2181 2888 3888 8181 8778
+EXPOSE 2181 2888 3888 8181 8778 8779
```
Comment (antban) [inline Dockerfile:60]: Can you please add port mapping here: https://github.com/zalando-nakadi/exhibitor-appliance/blob/master/exhibitor-appliance.yaml#L51

File: exhibitor.conf.tmpl [hunk 3]
```diff
@@ -11,3 +11,4 @@ election-port=3888
 zoo-cfg-extra=tickTime\=2000&initLimit\=10&syncLimit\=5&quorumListenOnAllIPs\=true
 auto-manage-instances-settling-period-ms=0
 auto-manage-instances=1
+java-environment=#!/bin/bash \nexport JAVA_TOOL_OPTIONS="-javaagent:/opt/jolokia-jvm-${JOLOKIA_VERSION}-agent.jar=port=8779,host=0.0.0.0"
```
Comment (antban) [inline exhibitor.conf.tmpl:14]: Can you please explain what is java-environment and how this thing is used? I failed to find documentation for it

Comment (ghost) [reply exhibitor.conf.tmpl:14]: The value of java-environment is placed in Config -> Ensemble -> java.env script. It is "Script to write as the “java.env” file which gets executed as part of the ZooKeeper start script. Optional – leave blank for no file." [docs](https://github.com/soabase/exhibitor/wiki/Configuration-UI)

**Comment by antban** [comment]
:+1:

**Comment by ghost** [comment]
@zrobgar can I ask you to review again?

**Comment by zrobgar** [comment]
👍

**Pull request closed without merging**